README.md

```diff
@@ -52,4 +53,4 @@
 #### Tasks
 - [ ] Learn Git/GitHub.
 - [ ]  Submit your Name for 100 Days of Code.
-> **IDIOT BOX** : Empty
\ No newline at end of file
+> **IDIOT BOX** : Empty
```

Comment:
Why did you change this line?
Also, you have to add a folder with your name.
Please read the README file again.